### PR TITLE
ceph-mgr packaging

### PR DESCRIPTION
**Pull request by tserong**
This adds a ceph-mgr package, with the caveat that it doesn't include Requires: lines in the spec file for the CherryPy, Django, gevent, python-dateutil and djangorestframework python packages (see the commit message in ee86291 for details). In the meantime, `pip2 install cherrypy django==1.8 gevent python-dateutil djangorestframework` will give you a runnable ceph-mgr.

### CODE REVIEW

**Comment by tserong** [comment]
@smithfarm, JFYI in case you want to review (although this is going into wip-ceph-mgr, not yet into master)

**Comment by smithfarm** [comment]
@tserong No deb packaging yet? ;-)

@ktdreyer @b-ranto Wanna have a look?

**Comment by tserong** [comment]
Hrm. It's possible I should have submitted this to @jcsp's wip-ceph-mgr branch, not ceph's wip-ceph-mgr branch, because then it would end up appending to #10328. Should I move this, or leave it here?

**Comment by jcsp** [comment]
We can leave it like this for the moment -- the ceph and jcsp branches are both in flux.  I've cherry-picked the commit from here that adds installation to CMakeLists to see if it helps with a mysteriously failure in the build on debian.

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -493,6 +494,24 @@ add_subdirectory(include)
 add_subdirectory(librados)
 add_subdirectory(libradosstriper)
 
+if (WITH_MGR)
```
Comment (tchaikov) [inline src/CMakeLists.txt]: better off adding another `CMakeLists.txt` in mgr, and moving this stuff into that file instead.

**Comment by jcsp** [comment]
Thank you -- I cherry picked the commits from here onto the wip-ceph-mgr branch before it merged to master

**Pull request closed without merging**